src/app/demo/index/index.component.html

```diff
@@ -4,20 +4,26 @@
     <slides-feedback-widget></slides-feedback-widget>
   </div>
 
-  <slides-slide id="index">
-    <h1>Angular Codelab</h1>
-    <h2>Welcome! Choose a topic below to start learning:</h2>
+  <div [ngSwitch]="firstVisit">
```

Comment:
ERROR in ng:///home/travis/build/AngularNYC/angular-presentation/src/app/demo/index/index.component.html (5,5): Property 'firstVisit' is private and only accessible within class 'IndexComponent'.